AppBuilder/platform/views/viewComponent/ABViewPivotComponent.js

```diff
@@ -71,6 +71,18 @@ module.exports = class ABViewPivotComponent extends ABViewComponent {
                   }
                },
             ],
+            [
+               pivot.views.table,
+               class CustomTable extends pivot.views.table {
+                  CellFormat(value) {
+                     const decimalPlaces = settings.decimalPlaces ?? 2;
```

Comment:
Is it better if we add the default value to `ABViewPivotCore` file [here](https://github.com/digi-serve/appbuilder_class_core/blob/v2/views/ABViewPivotCore.js#L10) ?

then use it to 
```const decimalPlaces = settings.decimalPlaces ?? baseView....[DEFAULT VALUE HERE];```